rules/manager_test.go

```diff
@@ -769,7 +769,7 @@ func reloadAndValidate(rgs *rulefmt.RuleGroups, t *testing.T, tmpFile *os.File,
 	_, err = tmpFile.Write(bs)
 	testutil.Ok(t, err)
 	err = ruleManager.Update(10*time.Second, []string{tmpFile.Name()}, nil)
-	testutil.Ok(t, err)
+	testutil.NotOk(t, err)
```

Comment:
Why does this fail now?

Reply:
Returned now is along with line and column number. I haven't been able to figure this out yet. Can you help me with this?

Reply:
Not offhand, it'd take debugging.

Reply:
@brian-brazil  When we marshal a struct containing the `yaml.Node` type, the []byte received contains additional information from the node. This is done in the `reloadAndValidate()`. This will create an error from the code since the output would be like:
```
groups:
  - name: test
    interval: 0s
    rules:
      - record:
            kind: 8
            style: 0
            tag: '!!str'
            value: job:http_requests:rate5m
            anchor: ""
            alias: null
            content: []
            headcomment: ""
            linecomment: ""
            footcomment: ""
            line: 4
            column: 15
        expr:
            kind: 8
            style: 0
            tag: '!!str'
            value: sum by (job)(rate(http_requests_total[5m]))
            anchor: ""
            alias: null
            content: []
            headcomment: ""
            linecomment: ""
            footcomment: ""
            line: 5
            column: 13
```
_Note: record and expr are `yaml.Node`._

Hence when we Parse this to rulefmt, this will return an error since `record:` and `expr:` are empty fields here and hence I used `testutil.NotOk()`

Reply:
That sounds like we need a better test then, as this one is no longer doing what it's meant to be doing

Reply:
I have fixed this by formatting the rules to the yaml v2 standards.